### PR TITLE
[i18n-<languageCode>] Translating agents.md to Chinese 

### PR DESCRIPTION
**Pull request by HMJ0628**
# What does this PR do?

I have translated agents.md to Chinese. Please let me know if there are any corrections. Thank you very much!

issue:[i18n-<languageCode>] Translating docs to Chinese Translate agents.md into Chinese #35135
Fixes # [(issue)](https://github.com/huggingface/transformers/issues/35135)

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [X] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by Rocketknight1** [comment]
cc @stevhliu

File: docs/source/zh/agents.md [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+>>> agent.run("Could you get me the title of the page at url 'https://huggingface.co/blog'?")
+
+(...)
+'Hugging Face – Blog'
+```
+
+如果有任何代码尝试执行非法操作，或者生成的代码出现常规 Python 错误，执行将停止。
+
+> [WARNING]
+> The LLM can generate arbitrary code that will then be executed: do not add any unsafe imports!
```
Comment (stevhliu) [inline docs/source/zh/agents.md:212]: Would you mind also translating the warning (just the content and not the syntax)?

```md
> [!WARNING]
> Chinese translation here
```

File: docs/source/zh/agents.md [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+```python
+from transformers import ReactJsonAgent
+from transformers.agents import PythonInterpreterTool
+
+agent = ReactJsonAgent(tools=[PythonInterpreterTool()], system_prompt="{your_custom_prompt}")
+```
+
+> [WARNING]
+> Please make sure to define the `<<tool_descriptions>>` string somewhere in the `template` so the agent is aware 
+of the available tools.
```
Comment (stevhliu) [inline docs/source/zh/agents.md:267]: This should also be translated

File: docs/source/zh/agents.md [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+
+当智能体初始化时，工具属性会用来生成工具描述，然后将其嵌入到智能体的系统提示中，这让智能体知道可以使用哪些工具以及为什么使用它们。
+
+### 默认工具箱
+
+Transformers 提供了一个默认工具箱，用于增强智能体，您可以在初始化时通过 `add_base_tools=True` 参数将其添加到智能体中：
+
+- **文档问答**：给定一个文档（如图像格式的 PDF），回答关于该文档的问题([Donut](./model_doc/donut))
+- **图像问答**：给定一张图片，回答关于该图像的问题([VILT](./model_doc/vilt))
+语音转文本：给定一个人讲述的音频录音，将其转录为文本（Whisper）
```
Comment (stevhliu) [inline docs/source/zh/agents.md:290]:
```suggestion
- 语音转文本：给定一个人讲述的音频录音，将其转录为文本（Whisper）
```

File: docs/source/zh/agents.md [hunk 4]
```diff
@@ -0,0 +1,428 @@
...
+
+该函数需要：
+- 一个清晰的名称。名称通常描述工具的功能。由于代码返回某个任务中下载次数最多的模型，因此我们将其命名为 `model_download_tool`。
+- 对输入和输出进行类型提示
+- 描述，其中包括 "`Args`:" 部分，描述每个参数（这次不需要类型指示，它会从类型提示中获取）。 
+
+所有这些将自动嵌入到智能体的系统提示中，因此请尽量使它们尽可能清晰！
+
+> [TIP]
+> This definition format is the same as tool schemas used in `apply_chat_template`, the only difference is the added `tool` decorator: read more on our tool use API [here](https://huggingface.co/blog/unified-tool-use#passing-tools-to-a-chat-template).
```
Comment (stevhliu) [inline docs/source/zh/agents.md:351]: The tip hould also be translated

**Review by stevhliu — COMMENTED** [review]
Very nice, thank you! As a reminder, you'll also need someone to review the translated content (I can't read Chinese) 🤗

File: docs/source/zh/agents.md (hunk 4 above)
Comment (STF-Zero) [reply docs/source/zh/agents.md]: The translation works for me!
It's of real convenience to get the contents.

**Comment by stevhliu** [comment]
Hi, no need to keep updating your branch!

**Comment by HMJ0628** [comment]
Should I pull request again？

**Comment by stevhliu** [comment]
You don't have to do anything! 🤗  Let the CI test below finish running and then I'll merge it for you!

**Review by stevhliu — COMMENTED** [review]
Hello, can you add `agents` to the toctree please?

**Comment by stevhliu** [comment]
Great! Last step is to remove the `transformers_agents.md` file (this is outdated now) and replace it in the toctree with your new `agents` file

**Comment by stevhliu** [comment]
> Last step to resolve the error is to remove the transformers_agents.md file (this is outdated now)

You still need to remove the transformers_agents.md file https://github.com/HMJ0628/transformers/blob/main/docs/source/zh/transformers_agents.md from your branch

**Comment by HMJ0628** [comment]
sorry, I have removed the file.

**Review by stevhliu — APPROVED** [review]
Thanks for your translation! 🤗

**Pull request merged**